File: pgm/ABC_Java/ABC_233_A.java
import java.io.*;
// import java.math.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final String[] arr = reader.readLine().split (" ");
        final double X = Double.parseDouble (arr[0]);
        final double Y = Double.parseDouble (arr[1]);
        System.out.println (
            String.format ("%.0f", Math.max (0, Math.ceil ((Y - X) / 10))));
        return;
    }
}

File: pgm/ABC_Kotlin/ABC_233_A.kt
// import java.io.BufferedOutputStream
// import java.io.BufferedReader
// import java.io.DataInputStream
// import java.io.FileInputStream
// import java.io.IOException
// import java.io.InputStreamReader
// import java.io.PrintWriter
// import java.lang.StringBuilder
// import java.time.LocalTime
// import java.time.format.DateTimeFormatter
// import java.util.Scanner
// import java.util.StringTokenizer

import kotlin.math.*

const val nine: Int = 9
const val ten: Int = 10
const val zero: Int = 0
const val space: String = " "

fun main() {
    val (X: Int, Y: Int) = readln().split(space).map { it.toInt() }
    println(max(zero, (Y - X + nine) / ten))
    kotlin.system.exitProcess(0)
}